documentation/navigation/deep-linking.md

```diff
@@ -0,0 +1,45 @@
+# Deep linking
+
+In the [explicit navigation](https://bumble-tech.github.io/appyx/navigation/explicit-navigation/) section we covered
```

Comment:
Don't link to full url directly, but to relative markdown files. Mkdocs will convert them automatically (also please do check them they work and look right with a local mkdocs run).

```suggestion
In the [explicit navigation](explicit-navigation.md) section we covered
```